bigbluebutton-html5/private/config/settings.yml

```
@@ -68,6 +68,13 @@ public:
     # https://github.com/bigbluebutton/bigbluebutton/pull/10826
     customHeartbeat: false
     showAllAvailableLocales: true
+    # Show "Audio Filters for Microphone" option in settings menu.
+    # When set to true, users are able to enable/disable microphone constraints,
+    # otherwise default values for 'microphoneConstraints' option
+    # are used.
+    # For more info, see 'microphoneConstraints' option in this config.
+    # If not set, default value is true.
+    # showAudioFilters: true
```

```suggestion
    showAudioFilters: true
```

All looks good, just let's include the parameter, please uncomment it